Sources/SFSafeSymbols/Symbols/SFSymbol+CaseIterable.swift

```diff
@@ -0,0 +1,6967 @@
+// Don't touch this manually, this code is generated by the SymbolsGenerator helper tool
+
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+extension SFSymbol: CaseIterable {
+    public static var allCases: [SFSymbol] {
+        if #available(iOS 14.2, macOS 11.0, tvOS 14.2, watchOS 7.1, *) {
```

Comment:
It would be great to add a `// MARK: - iOS 14.2` line (or something similar) before each `if #available` to simplify browsing the file.

Reply:
We can also break this massive if-else into small pieces (and reduce duplication):

```swift
private extension SFSymbol {
    @available(...)
    static let v1: [SFSymbol] = [
        ...
    ]

    @available(...)
    static let v2: [SFSymbol] = [
        ...
    ]

    @available(...)
    static let v2_1: [SFSymbol] = [
        ...
    ]
}

@available(...)
extension SFSymbol: CaseIterable {
    public static var allCases: [SFSymbol] {
        var result = SFSymbol.v1
        if #available(...) {
            result += SFSymbol.v2
        }
        if #available(...) {
            result += SFSymbol.v2_1
        }
        return result
    }
}
```

Reply:
@ddddxxx I like the idea, but this would probably lead to runtime performance regressions because the array gets rebuilt on every call, right?

 There is also probably no option for a `lazy static var`, or am I missing something here?

I also don't think the compiler would be able to optimize it due to the `#available` checks :(